lib/puppet/type/ini_setting.rb

```diff
@@ -132,6 +131,8 @@ def insync?(current)
     defaultto false
   end
 
+  # rubocop:enable GetText/DecorateString
```

Comment:
I cannot see this disabled in the file, therefore it will not need to be enabled.